targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/spi_api.c

```diff
@@ -44,21 +44,27 @@
 #include "pinmap_ex.h"
 #include "PeripheralPins.h"
 
-#include "nrf_drv_spi.h"
```

Comment:
does that break old SDKs?

Reply:
The new header ```nrfx_spi.h``` replaces ```nrf_drv_spi.h```.  Ultimately though there is no backward compatibility in our move to SDK 15.  The Nordic SDK includes a legacy folder (https://github.com/TacoGrandeTX/mbed-os/tree/denmark/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_15_0/integration/nrfx/legacy) but we've only carried forward the few files that are required for Mbed.    

I've just reviewed the files there and most of them can go away, so thanks for bringing this up.

Reply:
@paul-szczepanek-arm I've just removed the unnecessary legacy files with https://github.com/TacoGrandeTX/mbed-os/commit/833ed6392892b924ec9d3d6b8f7aae0a43746b56